### DISCUSSION TITLE
Clarify which files get installed when running `quarto use template` and what the role of `.quartoignore` is?

### DISCUSSION THREAD

**Discussion by ramiromagno** [quarto-dev/quarto-cli#6292] [Q&A]
### Description

I noticed that the files `README.md` and `LICENSE` do not get installed when I run `quarto use template`. Funnily enough, because I had a `README.qmd`, it got installed :) 

So do I understand it correctly that you have special exclusion rules for particular files? Isn't that being too smart here? Wouldn't it be better to leave that to be listed in `.quartoignore`? Do I interpret it correctly that `.quartoignore` should have a role similar to `.Rbuildignore` for R packages?

**Comment by mcanouil** [comment]
`README.md` and `LICENSE` files are files very common which are only there to describe the repository.
There are absolutely no reasons for them to be installed, either by `quarto add` or `quarto use`.
`README.qmd` is not a standard file to be in a repository thus Quarto has no reason to exclude it.
`.quartoignore` is a "ignore" file, as `.Rbuildignore` or  `.gitignore.

See <https://quarto.org/docs/extensions/starter-templates.html#creating-a-template>.

What else do you need? or what is not clear in the documentation?

**Reply by ramiromagno** [reply]
I understand that that is an opinionated design choice. I am just expressing a different opinion: I'd rather have `.quartoignore` being automatically added when I run:

```
quarto create extension journal
```

with `.quartoignore` populated with:

```
LICENSE
README.*
```

**Reply by mcanouil** [reply]
I believe the choice was "don't add files" unless necessary.
For instance, I often use "ignore everything but"  kind of `.gitignore` file, the fact that Quarto does write into it is annoying (https://github.com/quarto-dev/quarto-cli/issues/6011).

This being said, if Quarto writes to `.gitignore`, maybe it should do the same for `.quartoignore`.

In some aspect maybe a "quarto vaccinate" (see https://usethis.r-lib.org/reference/git_vaccinate.html) could serve to opt-in to create `*ignore` files.
